fdbclient/include/fdbclient/BlobGranuleCommon.h

```diff
@@ -408,4 +408,12 @@ struct BlobRestoreArg {
 	}
 };
 
+struct BlobGranuleBackupConfig : public KeyBackedClass {
+	BlobGranuleBackupConfig(KeyRef prefix = "\xff\x02/bgbackup/"_sr) : KeyBackedClass(prefix) {}
```

Comment:
Change `"\xff\x02/bgbackup/"_sr` to `SystemKey("\xff\x02/bgbackup/"_sr)` (which will work now after today's bug fix).  The argument type remains `KeyRef`, `SystemKey` is just a wrapper that

- makes sure we do not define system space prefixes that are nested within each other
- makes it easy to find system state constants while letting them be defined where they make sense, while also giving us the flexibility to instantiate system state in other prefixes for testing or other purposes.

Reply:
fixed. thank you!